godel_robots/abb/godel_irb2400/godel_irb2400_moveit_config/config/controllers.yaml

```diff
@@ -1,6 +1,5 @@
 controller_list:
-  - name: ""
-    action_ns: joint_trajectory_action
+  - name: 'joint_trajectory_action'
+    action_ns: ''
```

Comment:
On second thought, your script does set some environment variables required by ROS.  Doing this from within a ros launch file might be difficult so I'm OK with keeping the shell script as it makes it easier to configure the environment for running ROS across multiple machines.
What you can do is to put this script into its own ros package (e.g. godel_network_support ) so that you can use the rosrun command to call the script instead of having to cd into the source directory.